python/kfserving/setup.py

```diff
@@ -25,7 +25,7 @@
 
 setuptools.setup(
     name='kfserving',
-    version='0.2.1',
+    version='0.2.3',
```

Comment:
I released 0.2.2 with new code, but found a new issue that's fixed in this PR during integration, so updated new version :-) I do not think this is a big deal since the SDK may need to be 0.3 if cut 0.3 for kfserving or big changes :-)